ERCS/erc-7208.md

```diff
@@ -0,0 +1,598 @@
+---
+eip: 7208
+title: On-Chain Data Container
+description: Abstracting logic away from storage for RWA Tokenization
```

Comment:
If there's space, please expand "RWA" (and any other acronyms the first time they're used.)